server/db/models/song.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+var _ = require('lodash');
+var Sequelize = require('sequelize');
+
+module.exports = function (db) {
+
+     db.define('songs', {
+        title: {
+            type: Sequelize.STRING,
+            allowNull: false
+        },
+        description: {
+            type: Sequelize.TEXT,
+            allowNull: false
```

Comment:
Are we going to be writing descriptions for all the songs in our seed file or are we pulling those from somewhere? What kind of info do we expect to be in a song's description?